com.github.marinm.songrec.json

```
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.github.marinm.songrec",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "songrec",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host:ro",
+        "--filesystem=/tmp:ro",
+        "--filesystem=xdg-config",
+        "--filesystem=xdg-cache",
+        "--socket=pulseaudio"
+    ],
+    "build-options": {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/songrec/.cargo"
+        }
+    },
+    "modules": [
+        {
+            "name": "rust-flatpak",
```

```suggestion
            "name": "songrec",
```
(otherwise your CARGO_HOME wouldn't make sense as the build dirs are `/run/build/$moduleName`